sdk/face/ai-vision-face-rest/samples-dev/livenessWithVerifyWithImage.ts

```diff
@@ -0,0 +1,113 @@
+import { randomUUID } from 'crypto';
+
+import { AzureKeyCredential } from '@azure/core-auth';
+
+import createFaceClient, {
+    CreateLivenessWithVerifySession200Response,
+    DeleteLivenessWithVerifySession200Response,
+    GetLivenessWithVerifySessionAuditEntries200Response,
+    GetLivenessWithVerifySessionResult200Response,
+    CreateLivenessWithVerifySessionContentParametersPartDescriptor,
+    CreateLivenessWithVerifySessionContentVerifyImagePartDescriptor,
+    GetLivenessWithVerifySessions200Response,
+} from '@azure-rest/ai-vision-face';
+import { readFileSync } from 'fs';
+
+/**
+ * This sample demonstrates how to create a liveness detection session.
+ *
+ * @summary creates a liveness detection session
+ */
+
+const pressAnyKeyToContinue = () => new Promise<void>(resolve => {
+    const { stdin } = process;
+    const { isRaw } = stdin;
+    stdin.setRawMode(true);
+    stdin.resume();
+    stdin.once('data', () => {
+        stdin.pause();
+        stdin.setRawMode(isRaw);
+        resolve();
+    });
+});
+
+const waitForLivenessRequest = async () => {
+    // Wait for request from client device.
+};
+
+const sendTokenToClientDevices = async (token: string) => {
+    // Send the token to client devices.
+};
```

Comment:
I am not sure what this step is doing ?

Reply:
I am curious if the app server apis are also provided by Face Service? is there any machanism that can gurantee client side should only call step 6 and app server should only call step 2 and step 8? is this sample representing client or representing app server side ?

Reply:
App server should be implemented by customer, depending on their scenario. We provided another liveness check SDK for client side, while the communication between app server and client should also be implemented by customer. This sample representing app server side logic.